src/com/lilithsthrone/game/dialogue/npcDialogue/submission/RebelBaseInsaneSurvivorDialogue.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.lilithsthrone.game.dialogue.npcDialogue.submission;
+
+import com.lilithsthrone.game.character.npc.NPC;
+import com.lilithsthrone.game.dialogue.DialogueFlagValue;
+import com.lilithsthrone.game.dialogue.DialogueNode;
+import com.lilithsthrone.game.dialogue.responses.Response;
+import com.lilithsthrone.game.dialogue.responses.ResponseCombat;
+import com.lilithsthrone.game.dialogue.utils.UtilText;
+import com.lilithsthrone.main.Main;
+import com.lilithsthrone.utils.Util;
+import com.lilithsthrone.utils.Util.Value;
+
+/**
+ *
+ * @author dustland.crow
+ */
+public class RebelBaseInsaneSurvivorDialogue {
+    
+    private static NPC getAttacker() {
+		return Main.game.getActiveNPC();
+    }
+    
+    public static final DialogueNode INSANE_SURVIVOR_ATTACK = new DialogueNode("A Living Ghost", "A strange figure in the corner.", true) {
+        @Override
+        public String getAuthor() {
+                return "DSG";
+        }
+        @Override
+        public int getSecondsPassed() {
+                return 30;
+        }
+        @Override
+        public String getContent() {
+            return UtilText.parseFromXMLFile("encounters/submission/rebelBase/InsaneSurvivorAttack", "ATTACK", getAttacker());
+        }
+
+        @Override
+        public Response getResponse(int responseTab, int index) {
+            if (index == 1) {
+                return new ResponseCombat("Fight", "[npc.She] seems determined to fight you. Oblige [npc.herHim].",
+                        getAttacker(), 
+                        Util.newHashMapOfValues(
+                        new Value<>(getAttacker(), UtilText.parse(getAttacker(), "[npc.speech(I'll fucking end you, demon scum!)]"))));
+            }
+            else if (index == 2) {
+                return new Response("Talk", "You get the feeling [npc.she] is beyond reason but it might be worth a try.", INSANE_SURVIVOR_TALK_ATTEMPT);
+            }
+            else if (index == 3 && Main.game.getPlayer().getMoney() <= 0) {
+                return new Response("Offer money", "Your empty pockets don't give you the means to try this.", null);
+            }
+            else if (index == 3) {
+                return new Response("Offer money", "[npc.She] probably isn't after money it might be worth a try.", INSANE_SURVIVOR_BRIBE_ATTEMPT);
```

Comment:
Missing 'but', maybe change it so it's a little different from the 'Talk' option?
'While [npc.she] probably isn't after money it could be worth trying it.'

Reply:
Fixed